Modules/Scripted/SampleData/SampleData.py

```diff
@@ -856,51 +861,53 @@ def downloadFile(self, uri, destFolderPath, name, checksum=None):
         (algo, digest) = extractAlgoAndDigest(checksum)
         if not os.path.exists(filePath) or os.stat(filePath).st_size == 0:
             import urllib.request, urllib.parse, urllib.error
-            self.logMessage(f'<b>Requesting download</b> <i>{name}</i> from {uri} ...')
+            self.logMessage(_('Requesting download {name} from {uri} ...').format(name=name, uri=uri))
```

Comment:
Is the removal of `<b>` intentional ?

Reply:
Yes, it is part of `logMessage` method now (by default it is added around the text).